vms/platformvm/state/state.go

```
@@ -1447,7 +1447,11 @@ func (s *state) loadCurrentValidators() error {
 		}
 		tx, _, err := s.GetTx(txID)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed loading validator transaction txID %v, %w", txID, err)
```

moved slightly up. This should never ever fail, but always better fail fast